google-cloud-graalvm-samples/graalvm-samples-client-library/spanner-sample/src/main/java/com/example/InstanceOperations.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example;
+
+import com.google.cloud.spanner.Instance;
+import com.google.cloud.spanner.InstanceAdminClient;
+import com.google.cloud.spanner.InstanceConfigId;
+import com.google.cloud.spanner.InstanceId;
+import com.google.cloud.spanner.InstanceInfo;
+
+/**
+ * Helper methods to manage Spanner instances.
+ */
+public class InstanceOperations {
+
+  static void createTestInstance(
+      InstanceAdminClient instanceAdminClient, String projectId, String instanceId) {
+
+    if (instanceExists(instanceAdminClient, instanceId)) {
+      instanceAdminClient.deleteInstance(instanceId);
+    }
+
+    InstanceInfo instanceInfo =
+        InstanceInfo.newBuilder(InstanceId.of(projectId, instanceId))
+            .setInstanceConfigId(
+                InstanceConfigId.of(projectId, "regional-us-central1"))
```

Comment:
Might want to someday extract this into a property.

Reply:
Noted - for now since this is an emulator-based test the region string doesn't changes things so I will just leave it for now.